homeassistant/components/vicare/sensor.py

```diff
@@ -221,6 +270,150 @@ class ViCareSensorEntityDescription(SensorEntityDescription, ViCareRequiredKeysM
         device_class=SensorDeviceClass.ENERGY,
         state_class=SensorStateClass.TOTAL_INCREASING,
     ),
+    ViCareSensorEntityDescription(
+        key=SENSOR_GAS_SUMMARY_CONSUMPTION_CURRENT_DAY,
```

Comment:
IMO, if they are single-use, I don't think creating the constants is very useful.
```suggestion
        key="gas_summary_consumption_heating_currentday",
```

The same would apply to all keys...